src/runtime/errors.ts

```diff
@@ -257,6 +263,7 @@ export var messages = {
 
   unimplementedExpression: (op) => `There's no definition for the function ${op}`,
 
+  blankScan: () => 'Searching a lookup requires at least one of record, attribute, value, or node to be present',
```

Comment:
I would change the error to:

"lookup requires at least one attribute: record, attribute, value, or node"